### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by bzsolt**
JSRemoteTest-DCO-1.0-Signed-off-by: Zsolt Borbély zsborbely.u-szeged@partner.samsung.com

### CODE REVIEW

File: API/builder/targets/artik053.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def _build_iotjs(self, profile, extra_flags):
 
     def _append_testfiles(self):
         '''
-        Add test files to the ROMFS file of NuttX.
+        Add test files to the ROMFS file of TizenRT.
```
Comment (rtakacs) [inline API/builder/targets/artik053.py]: This comment is only true for the NuttX target (where the default ROMFS image is overwritten). Instead, I would add the following comment:
Add test files to the content folder of TizenRT.

Comment (bzsolt) [reply API/builder/targets/artik053.py]: Fixed.

**Comment by rtakacs** [comment]
LGTM

**Comment by yichoi** [comment]
LGTM

**Pull request merged**